app/models/reviewable_claimed_topic.rb

```diff
@@ -7,11 +7,12 @@ class ReviewableClaimedTopic < ActiveRecord::Base
 
   def self.claimed_hash(topic_ids)
     result = {}
-    if SiteSetting.reviewable_claiming != "disabled"
+    if SiteSetting.reviewable_claiming == "disabled"
       ReviewableClaimedTopic
-        .where(topic_id: topic_ids)
-        .includes(:user)
-        .each { |rct| result[rct.topic_id] = rct.user }
+        .where(topic_id: topic_ids, system: true)
+        .each { |rct| result[rct.topic_id] = rct }
+    else
+      ReviewableClaimedTopic.where(topic_id: topic_ids).each { |rct| result[rct.topic_id] = rct }
```

Comment:
In this case, should we looks only for claimed topics where system is false?

Reply:
Nope, it needs to be all of them in this case: the hash that this method returns is used to show in the UI which reviewables are claimed. When `reviewable_claiming` is disabled, the UI should only show system claims as being claimed. When it's enabled, the UI should show all claims as being claimed.